tests/unittests/utils/test_repo_normalize.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Zilliz. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+from towhee.utils.repo_normalize import RepoNormalize
+
+rn1 = RepoNormalize('resnet-image-embedding?ref=my-tag&model=resnet50#operator')
+rn2 = RepoNormalize('test/resnet-image-embedding[pytorch]')
+rn3 = RepoNormalize('resnet-image-embedding')
+rn4 = RepoNormalize('resnet_image_embedding')
+
+
+class TestRepoNormalize(unittest.TestCase):
+    """
+    Unittest for repo normalize.
+    """
+
+    def test_parse_uri(self):
+        result = rn1.parse_uri()
+        self.assertEqual(result.full_uri, 'https://towhee.io/towhee/resnet-image-embedding?ref=my-tag&model=resnet50#operator')
+        self.assertEqual(result.author, 'towhee')
+        self.assertEqual(result.repo, 'resnet-image-embedding')
+        self.assertEqual(result.ref, 'my-tag')
+        self.assertEqual(result.repo_type, 'operator')
+        self.assertEqual(result.scheme, 'https')
+        self.assertEqual(result.netloc, 'towhee.io')
+        self.assertEqual(result.query, {'model': 'resnet50'})
```

Comment:
the following properties should be added:
1. result.module_name
2. result.class_name
3. result.function_name

Reply:
What's the function_name? And is there any format for module_name?